Makefile

```
@@ -110,3 +110,13 @@ send-api-req-prod: ## Send request to production
 .PHONY: db-shell
 db-shell: ## Opens the mysql shell inside the db container
 	docker compose exec db bash -c 'mysql -uopenpodcast -popenpodcast openpodcast'
+
+.PHONY: db-init-auth
+db-init-auth: ## Initialize the auth db after api is up
+	# creates the auth schema
+	docker cp ./db_schema/auth.sql api-db-1:/tmp/auth.sql
+	docker compose exec db bash -c 'mysql -uopenpodcast -popenpodcast openpodcast_auth < /tmp/auth.sql'
+	# creates some dummy auth data 
+	docker cp ./db_auth_data.sql api-db-1:/tmp/auth_data.sql
+	docker compose exec db bash -c 'mysql -uopenpodcast -popenpodcast openpodcast_auth < /tmp/auth_data.sql'
```

Maybe we should not depend on the DB container name? It's auto-generated after all. If the service restarts, it might have a different name.

```suggestion
.PHONY: db-init-auth
db-init-auth: ## Initialize the auth db after api is up
	# Get the actual container name dynamically
	DB_CONTAINER=$$(docker compose ps -q db)
	
	# creates the auth schema
	docker cp ./db_schema/auth.sql $$DB_CONTAINER:/tmp/auth.sql
	docker compose exec db bash -c 'mysql -uopenpodcast -popenpodcast openpodcast_auth < /tmp/auth.sql'
	# creates some dummy auth data 
	docker cp ./db_auth_data.sql $$DB_CONTAINER:/tmp/auth_data.sql
	docker compose exec db bash -c 'mysql -uopenpodcast -popenpodcast openpodcast_auth < /tmp/auth_data.sql'auth_data.sql'
```

---

variables can't be used as every line is executed in its own shell instance. but I changed it to a flexible version.